weblab/config/settings/dev.py

```diff
@@ -80,3 +80,34 @@
     app.split('.')[0]: copy.deepcopy(local_logger_conf)
     for app in LOCAL_APPS
 })
+
+
+class InvalidStringShowWarning(str):
+    def __mod__(self, other):
+        import logging
+        logger = logging.getLogger(__name__)
+        logger.warning("In template, undefined variable or unknown value for: '%s'" % (other,))
+        return ""
+
+
+TEMPLATES = [
```

Comment:
Instead of putting the whole `TEMPLATES` definition here, can you just set the single relevant option? i.e.
```python
TEMPLATES[0]['OPTIONS']['string_if_invalid'] = InvalidStringShowWarning("%s")
```